src/Db/Dump/PKSimDB_dump.txt

```diff
@@ -3877,7 +3877,7 @@ INSERT INTO tab_calculation_method_rates VALUES('Lumen_PKSim','PARAM_Area','PI*(
 INSERT INTO tab_calculation_method_rates VALUES('Lumen_PKSim','PARAM_DrugAbsorption_para','OralApplicationsEnabled ? P_int_para*Aeff*(EffectiveLumenConcentration-(NOT SinkCondition_para ? fu*DrugMucosa : 0)) : 0','Amount per time');
 INSERT INTO tab_calculation_method_rates VALUES('Lumen_PKSim','PARAM_DrugAbsorption_trans','OralApplicationsEnabled ? P_int_trans*Aeff*(P_int_trans_lum_cell_factor*EffectiveLumenConcentration-(NOT SinkCondition_trans ? P_int_trans_cell_lum_factor*fu/K_cell_pls*DrugMucosa : 0)) : 0','Amount per time');
 INSERT INTO tab_calculation_method_rates VALUES('Lumen_PKSim','PARAM_EffectiveSurfaceArea','Ageom*AeffFactor*AeffVariabilityFactor','Area');
-INSERT INTO tab_calculation_method_rates VALUES('Lumen_PKSim','PARAM_FillLevel','Liquid/V','Dimensionless');
+INSERT INTO tab_calculation_method_rates VALUES('Lumen_PKSim','PARAM_FillLevel','V>0 ? Liquid/V : 0','Dimensionless');

```

Comment:
the problem was with the cattle simulation (every simulation, not just pop simulation).
Becasue the cattle GI-tract is not properly implemented yet, all GI parameters are set to zero, resulting in zero volume of all GI compartments.
This resulted in some 0/0 terms of the ODE and thus in NaNs in the RHS.

Before the SimModel fix those terms were apparently not properly evaluated (returning 0 or whatever else).

Adding the check for zero volume here and in the formulas below solves the issue.

Reply:
crazy

Reply:
Actually, probably not 0/0 but "Infinity" (as in the example above). Anyways :)